src/Apps/Comparables/index.tsx

```diff
@@ -0,0 +1,81 @@
+import { Box, Spinner } from "@artsy/palette"
```

Comment:
This is the main view which requests an artwork, and then sets up the filter with the artwork attributes.

This just accepts an `artworkID` (from Storybooks) for now, but next this will basically be its own text box where you can paste in an artwork id and click a button to see the results.